src/whylogs/app/logger.py

```diff
@@ -299,6 +299,8 @@ def log(
         features: Optional[Dict[str, any]] = None,
         feature_name: str = None,
         value: any = None,
+        character_list: str = None,
```

Comment:
should the character list be tied to the profile/logger? It seems wrong to have this change across different `.log` call on the same logger?

Reply:
yeah it needs to stay the same otherwise there is an warning in string tracker. I can update to raise an error.

Reply:
This logic is already part of the original string tracker PR.